client/Review.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import WouldRecommend from './WouldRecommend.jsx';
+import WouldNotRecommend from './WouldNotRecommend.jsx';
+import Verified from './Verified.jsx';
+import moment from 'moment';
+
+function Review({ review }) {
+  return <li className="review">
+    <div className="main">
```

Comment:
when using css modules i think you import the stylesheet into your component files and link to it per item(may be wrong), but you talked about using styled components this morning

Reply:
Hey, Andrew! I'm actually going to be switching over to styled components but either way this is just using a global style sheet so I kinda jumped the gun making this PR.